### PR TITLE
Swift Build Script | CD release pipeline

### PR DESCRIPTION
**Pull request by CyonAlexRDX**

### CODE REVIEW

File: src/core/types/decimal.rs [hunk 1]
```diff
@@ -11,26 +11,26 @@ use radix_engine_common::math::Decimal as ScryptoDecimal;
     DeserializeFromStr,
     uniffi::Record,
 )]
-pub struct Decimal {
+pub struct Decimal192 {
```
Comment (micbakos-rdx) [inline src/core/types/decimal.rs:14]: @CyonAlexRDX Why is this needed?

Comment (CyonAlexRDX) [reply src/core/types/decimal.rs:14]: Clash with Swift Foundations type with same name. So in iOS app in a file where we import
Both Sargon and Foundation it does not compile. And I don't wanna have to write Sargon.Decimal all over the place.
